src/app/services/baw-api/security/security.service.spec.ts

```diff
@@ -213,18 +213,15 @@ describe("SecurityService", () => {
         );
       }
 
-      it("should call signOut", async () => {
+      // If the user is logged out before logging in, the authentication token
+      // will be rotated.
+      // We used to do this before we started rotating authentication tokens,
+      // so this test protects against regressions.
+      it("should not sign the user out", async () => {
```

Comment:
```suggestion
      it("should NOT sign the user out", async () => {
```

This is a personal style thing: whenever I convert a test to a counter test (invert the behaviour), I capitalize the NOT to emphasize the new behaviour. 

Your comment is even better than my little style qurik